dev-packages/node-integration-tests/jest.setup.js

```diff
@@ -1,7 +1,7 @@
 const { cleanupChildProcesses } = require('./utils/runner');
 
-// Increases test timeout from 5s to 45s
-jest.setTimeout(45000);
+// Default timeout: 15s
+jest.setTimeout(15000);
```

Comment:
Reducing this by default, in cases where we need more we can opt-in to a higher timeout in a case-by-case basis (which also leads to us being conscious about which tests take long)